operations/commit_queue_test.go

```diff
@@ -142,6 +143,37 @@ func (s *CommitQueueSuite) TestListContentsForCLI() {
 	s.Contains(stringOut, patchURL)
 }
 
+func (s *CommitQueueSuite) TestMergeWithEmptyPatch() {
+	s.Require().NoError(db.ClearCollections(commitqueue.Collection, patch.Collection, model.ProjectRefCollection))
```

Comment:
do we need to clear the `commitqueue` collection?

Reply:
It is not! thanks you!